vanilla/vanilla
# Fix padding on wysiwyg post textbox.

Closes #9249 

The side padding on the Advanced Editor's Wysiwyg posting textbox is too thin. This PR adds some padding to make it look better.

### TO TEST
Enable Advanced Editor and choose Wysiwyg posting format. 
Go to post a comment. 
Notice that there is no space between the box's edge and the text you are going to type.

Then check out this branch and go to post a comment.
Notice that this is some space between the box's edge and the text you are going to type.
